pint/observatory/topo_obs.py

```diff
@@ -122,6 +127,16 @@ def gps_fullpath(self):
         return os.path.join(os.getenv('TEMPO2'),'clock',fname)
 
     @property
+    def bipm_fullpath(self):
+        """Returns full path to the TAI TT(BIPM) clock file.  Will first try PINT
+        data dirs, then fall back on $TEMPO2/clock."""
+        fname = 'tai2tt_bipm2015.clk'
```

Comment:
If we could have the files updated and cached when run that would be better but I'm not sure how the clock correction files are made and if they are stored anywhere in the correct format.